### DISCUSSION TITLE
Expsoing Dashboards/Map to Public?

### DISCUSSION THREAD

**Discussion by TomBayne** [telekom-security/tpotce#1206] [General]
Just got this setup - wondering if it's safe to expose the map to the public internet via reverse proxy?

Thanks :)

**Comment by TomBayne** [comment]
So after some poking around this seems pretty much safe to do. However I can't figure out how to actually disable HTTP auth just for this one page. Where is the config file I need to edit?

**Reply by t3chn0m4g3** [reply]
Authentication is bound to the web root. You can see here `docker/nginx/dist/conf/tpotweb.conf`, you can adjust the file to your needs by following the guide to volumes in the Wiki.
